tools/cli/domainCommands.go

```diff
@@ -516,9 +550,15 @@ type DomainMigrationRow struct {
 }
 
 type ValidationDetails struct {
-	CurrentDomainRow       *types.DescribeDomainResponse
-	NewDomainRow           *types.DescribeDomainResponse
-	LongRunningWorkFlowNum *int
+	CurrentDomainRow        *types.DescribeDomainResponse
+	NewDomainRow            *types.DescribeDomainResponse
+	LongRunningWorkFlowNum  *int
+	MismatchedDynamicConfig MismatchedDynamicConfig
```

Comment:
We might have more than one mismatched configs
```suggestion
	MismatchedDynamicConfigs []MismatchedDynamicConfig
```